src/processor/operator/persistent/reader/csv/driver.cpp

```diff
@@ -663,6 +663,10 @@ void copyStringToVector(ValueVector* vector, uint64_t rowToAdd, std::string_view
         if (!utf8proc::Utf8Proc::isValid(strVal.data(), strVal.length())) {
             throw common::CopyException{"Invalid UTF8-encoded string."};
         }
+        if (strVal.length() > BufferPoolConstants::PAGE_4KB_SIZE) {
+            throw CopyException(
+                ExceptionMessage::overLargeStringValueException(std::to_string(strVal.length())));
```

Comment:
Let's refactor this function to take a `size_t` or `uint64_t`.

Reply:
And on that note, is this the only call site? If so, inline it and delete the function.

Reply:
Should've checked.. It's not the only call site

Reply:
Added a check function.